src/applications/edu-benefits/1990/config/form.js

```diff
@@ -102,6 +103,7 @@ const formConfig = {
   },
   title: 'Apply for education benefits',
   subTitle: 'Form 22-1990',
+  footerContent: FormFooter,
   getHelp: GetFormHelp,
```

Comment:
Seems like we could remove getHelp in these configs?

Reply:
Hm, not sure I totally understand. The new `FormFooter` component checks if anything is specified for `getHelp` and if so renders what has been passed to it. Some forms don't include additional getHelp text in the config, so I think we still need to pass it to the config where we specifically want it.

Reply:
Oh, right, I see. I think we'll want to change that later; I'd rather just have a different component for forms that use GetHelp vs those that don't. You don't need to change that in here, though, I can make a ticket to refactor that later.

Reply:
ah gotcha, that makes sense